src/functions/editEmbed.js

```diff
@@ -0,0 +1,59 @@
+module.exports = async (d) => {
+  const data = d.util.aoiFunc(d);
+  var [
+    channelID = d.channel?.id,
+    messageID = d.message?.id,
+    index = 1,
+    option = "description",
```

Comment:
Remove default option, probably not the best to have this especially when some embeds dont have a description